src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -332,168 +337,38 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
-                    symbol={reserve.symbol}
-                    incentives={reserve.vIncentivesData}
-                    displayBlank={true}
-                  />
-                </PanelItem>
-                {reserve.stableBorrowRateEnabled && (
-                  <PanelItem
-                    title={
-                      <StableAPYTooltip
-                        text={<Trans>APY, stable</Trans>}
-                        key="APY_res_stable_type"
-                        variant="description"
-                      />
-                    }
-                  >
-                    <FormattedNumber value={reserve.stableBorrowAPY} percent variant="main16" />
-                    <IncentivesButton
-                      symbol={reserve.symbol}
-                      incentives={reserve.sIncentivesData}
-                      displayBlank={true}
-                    />
-                  </PanelItem>
-                )}
-                {reserve.borrowCapUSD && reserve.borrowCapUSD !== '0' && (
-                  <PanelItem title={<Trans>Borrow cap</Trans>}>
-                    <FormattedNumber value={reserve.borrowCap} variant="main16" />
-                    <ReserveSubheader value={reserve.borrowCapUSD} />
-                  </PanelItem>
-                )}
-              </Box>
-              {renderCharts && (
-                <ApyGraphContainer
-                  graphKey="borrow"
+                      <Trans>More details</Trans>
+                    </Link>
+                  </Trans>
+                </Warning>
+                <BorrowInfo
                   reserve={reserve}
                   currentMarketData={currentMarketData}
+                  currentNetworkConfig={currentNetworkConfig}
+                  renderCharts={renderCharts}
+                  showBorrowCapStatus={showBorrowCapStatus}
+                  borrowCap={borrowCap}
                 />
-              )}
-              <Box
-                sx={{ display: 'inline-flex', alignItems: 'center', pt: '42px', pb: '12px' }}
-                paddingTop={'42px'}
-              >
-                <Typography variant="subheader1" color="text.main">
-                  <Trans>Collector Info</Trans>
-                </Typography>
               </Box>
-              {currentMarketData.addresses.COLLECTOR && (
-                <ReserveFactorOverview
-                  collectorContract={currentMarketData.addresses.COLLECTOR}
-                  explorerLinkBuilder={currentNetworkConfig.explorerLinkBuilder}
-                  reserveFactor={reserve.reserveFactor}
-                />
-              )}
-            </Box>
+            ) : (
+              <BorrowInfo
+                reserve={reserve}
+                currentMarketData={currentMarketData}
+                currentNetworkConfig={currentNetworkConfig}
+                renderCharts={renderCharts}
+                showBorrowCapStatus={showBorrowCapStatus}
+                borrowCap={borrowCap}
+              />
```

Comment:
I think the ternary doesn't need to wrap the `<BorrowInfo />` component since it's duplicated.  It looks like it only needs to wrap the warning.  Let's de-dupe this.

Reply:
Good catch, moved it outside

Reply:
👏 This file is getting much cleaner